redaxo/src/addons/mediapool/lang/de_de.lang

```diff
@@ -59,7 +59,7 @@ pool_file_deleted = Datei wurde gelöscht
 pool_files_deleted = {0} Dateien wurden gelöscht.
 pool_file_delete_error = Die Datei "{0}" kann nicht gelöscht werden!
 pool_kat_delete_error = Die Kategorie "{0}" kann nicht gelöscht werden!
-pool_object_in_use_by = Sie wird derzeit von folgenden Objekten verwendet:
+pool_object_in_use_by = Sie wird derzeit von folgenden Objekten verwendet: {0}
```

Comment:
Also nochmal hierzu: Mir ist nicht ganz klar, wieso du hier die Liste mit in die Message holst?
Denn ohne bräuchten wir ja auch das rawMsg nicht.

Reply:
Vorher wurden Fehlermeldungen ausserhalb der Löschprozesses abgewickelt, d.h. man hätte es sich jedes mal neu generieren müssen und bekam dann ein großes $return array Paket. Das läuft nun über api_exceptions und daher kommt nur eine Message raus. Müsste so bleiben, befürchte ich

Reply:
Naja, du könntest doch an der stelle statt:
`rex_i18n::rawMsg(..., $uses)`
auch das machen:
`rex_i18n::msg(...).' '.$uses`

Aber ist jetzt auch nicht soo wichtig. Dann lass ruhig so.
Mich stört immer rawMsg, da die Übersetzer das nicht ahnen können, dass die msg als rawMsg ausgegeben wird und somit htmlspecialchars selbst notiert werden müssen (`&gt;` etc.). Allerdings ist hier eher unwahrscheinlich, dass solche Sonderzeichen in die Übersetzungen gelangen werden.

Reply:
ist entsprechend umgebaut